Specs/DateTime/German/DateTimeExtractor.json

```diff
@@ -96,7 +96,7 @@
     ]
   },
   {
-    "Input": "Das ganze ist am 14. Oktober um 8:00:00 Uhr",
+    "Input": "Das ganze ist am 14. Oktober um 8:00:00 Uhr vorbei",
```

Comment:
Don't these two sentences mean different things?

Reply:
The initial one is not a complete sentence, so we added this word. The old sentence actually doesn't have a meaning as it lacks word(s).